tests/DotNetLightning.Core.Tests/Serialization.fs

```diff
@@ -124,7 +124,7 @@ module SerializationTest =
 
                     let unsignedChannelAnnoucement = {
                         Features = features
-                        ChainHash = if (not nonbitcoinChainHash) then uint256(hex.DecodeData("6fe28c0ab6f1b372c1a6a246ae63f74f931e8365e15a089c68d6190000000000")) else uint256(hex.DecodeData("000000000933ea01ad0ee984209779baaec3ced90fa3f408719526f8d77f4943"))

```

Comment:
`uint256(hex.DecodeData("6fe28c0ab6f1b372c1a6a246ae63f74f931e8365e15a089c68d6190000000000"))`

Equals to `NBitcoin.Network.Main.Consensus.HashGenesisBlock` which means, you shouldn't change this line.
Instead change the expected value in the line 148.